driver/src/order_driver.rs

```diff
@@ -94,6 +97,22 @@ fn update_balances(state: &mut State, orders: &[Order], solution: &Solution) {
     }
 }
 
+fn get_standing_orders_index(standing_orders: &Vec<models::StandingOrder>) -> Vec<u32> {
```

Comment:
My question is about what this function is doing? Is it trying to determine which (of all standing orders) is the correct one? Is the index not related to the length of the list of previous standing orders? I see there is a vector here? Shouldn't an index be a number? 

I must be missing something, could you please tell me what this is about?

Reply:
does the vector represent the index of standing orders where auction id corresponds to the return vector's index?

Reply:
This is the list of standing order indexes used in the solution calculation.

Reply:
It appears the code here has been changed substantially since I last asked my questions here. Looks much cleaner!